commit 4aead37f0149f3cf6bf44988cdd32b6ec1536fe1
Author: Christopher Friedt <chrisfriedt@gmail.com>
Date:   Wed Apr 27 23:52:43 2011 -0400

    added RUN_LVL,etc {set,get}utent definitions to utmp.h

diff --git a/libc/include/utmp.h b/libc/include/utmp.h
index ffd3c92..0f36642 100644
--- a/libc/include/utmp.h
+++ b/libc/include/utmp.h
@@ -79,12 +79,24 @@ struct utmp
 #define ut_time ut_tv.tv_sec
 #define ut_addr ut_addr_v6[0]
 
+/* Definitions for ut_type. */
+#define EMPTY              0
+#define RUN_LVL            1    /* this is a RUN_LEVEL record */
+#define BOOT_TIME          2    /* this is a REBOOT record */
+#define INIT_PROCESS       5    /* this process was spawned by INIT */
+#define LOGIN_PROCESS      6    /* this is a 'getty' process waiting */
+#define USER_PROCESS       7    /* any other user process */
+#define DEAD_PROCESS       8    /* this process has died (wtmp only) */
+
 __BEGIN_DECLS
 
 int utmpname(const char*);
 void setutent();
 struct utmp* getutent();
 
+void setutent(void);
+void endutent(void);
+
 __END_DECLS
 
 #endif /* _UTMP_H_ */

